tensorflow_app/views/import_graphdef.py

```diff
@@ -186,7 +192,10 @@ def import_graph_def(request):
                 layer['params']['dim'] = str(input_dim)[1:-1]
 
             elif layer['type'][0] == 'Convolution':
-                if str(node.name) == name + '/weights' or str(node.name) == name + '/kernel':
+                if str(node.name) in [name + '/weights', 'Repeat/' + name + '/weights',
+                                      name + '/kernel', 'Repeat/' + name + '/kernel',
+                                      name + '/weights', 'Stack/' + name + '/weights',
+                                      name + '/kernel', 'Stack/' + name + '/kernel']:
```

Comment:
Aren't `name+'/weights'` and `name+'/kernel'` written twice?

Reply:
My Bad, I'll remove it.